src/Chapter2.hs

```diff
@@ -740,7 +755,8 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Int -> [[Int]] -> [[Int]]
+contains n l = filter (elem n) l
```

Comment:
This is the correct and expected implementation :+1: 
Additionally, you can eta-reduce this function as well